src/_core.scss

```diff
@@ -0,0 +1,14 @@
+// GLOBAL SCOPE LOADER: CORE
+// This partial loads Core scope related Partials.
+// - This partials Generates Core custom properties using a mixin and a settings map.
+//   - Mixin is "generate-custom-properties".
+//   - Map is "$core-settings-map" which is a map collected from all other core scope maps.
+// - Core properties are Global and Generic and goes under :root
+
+@import "core/core-tokens";
```

Comment:
why do we need to import other files besides - core-settings

Reply:
- We use tokens to registraion values.
- we use map to key value tokens
- we use settings to collect the maps and generate the custom properties from the maps.

- maps have tokens dependencies.
- setting have map dependencies.
- i want to manage all dependencies in a loder (one place).
- This is a convention for global scope CSS.